tools/get_release_contributors.py

```diff
@@ -89,36 +96,36 @@ def get_login_or_realname(self, key, commit_hash):
         login = self._get_login(key, commit_hash)
         if login:
             return f'@{login}'
-        m = re.match('\A([^@<>]+) <.*>\Z', key)
-        if m:
-            return m.group(1)
-        logger.warning(f'unable to extract github login or real name from {repr(key)}')
+        matches = re.match(r'\A([^@<>]+) <.*>\Z', key)
+        if matches:
+            return matches.group(1)
+        logger.warning("unable to extract GitHub login or real name from %r", key)
```

Comment:
Why do you change away from a format string here but above you changed to? Or does the logger support the key as something special.

Reply:
As per other comment - logging is a special case